tests/rptest/tests/restart_services_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import requests
+
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.admin import Admin
+from rptest.services.redpanda import ResourceSettings, LoggingConfig, SchemaRegistryConfig
+
+log_config = LoggingConfig('info',
+                           logger_levels={
+                               'admin_api_server': 'trace',
+                               'security': 'trace',
+                               'pandaproxy': 'trace',
+                               'kafka/client': 'trace'
```

Comment:
I can see security log being noise but I figured we would want logs on the Schema Registry (panda proxy logger) and kafka client since those are reset in `api::restart`

Reply:
Up to you. Better to err on the side of caution.

Reply:
Done in https://github.com/redpanda-data/redpanda/pull/8127/commits/747d13a8678e7ea628a3972add8686d2b2568b77